src/scipp/visualization/colors.py

```diff
@@ -27,7 +28,9 @@ def rgb_to_hex(rgb):
     return "#" + "".join(hex_value)
 
 
-def make_random_color(fmt='rgb'):
+def make_random_color(
```

Comment:
Is this still used? Remnant from plotting in Scipp, pre-dating Plopp?

Reply:
No, it isn't. I removed it now.